### ISSUE TITLE
Add documentation of minimum Alicloud RAM permission for packer plugin

### ISSUE THREAD

**Issue by binkesi** [hashicorp/packer-plugin-alicloud#85]
Hi team, when I try to use packer plugin for Alicloud, I found I need some specific RAM permission to call Alicloud APIs. It hasn't been figure out in this doc: https://developer.hashicorp.com/packer/plugins/builders/alicloud. I think it will be clearer after add this part of doc. Here is my PR:
https://github.com/hashicorp/packer-plugin-alicloud/pull/84

**Comment by JadeLi1** [comment]
Yes, the document is quite confused on the credential, especially the `ram_role_name` which appears in both required section and optional section.  Seems 2 different ram roles should be used.

**Comment by nywilken** [comment]
Closing since #84 has been merged. If there are still confusions around the documentation once released please leave a comment on this thread and I will gladly reopen.
